### DISCUSSION TITLE
Modifying Camera Settings At Runtime

### DISCUSSION THREAD

**Discussion by NecrosisDev** [Sixze/ALS-Refactored#417] [Q&A]
How would I go about tweaking individual post process weights, modifying field of view, or swapping camera setting data assets at runtime? I've tried a few methods but setting variables in the structs doesn't work (read only), and I can't seem to set the camera settings data asset outside of the character blueprint.

**Comment by ameaninglessname** [comment]
IMO, The built-in camera system of ALS is kind of `static`, the design itself is powerful and unique though.
You could try to find alternatives for that on [marketplace](https://www.unrealengine.com/marketplace/en-US/content-cat/assets/codeplugins?count=20&sortBy=effectiveDate&sortDir=DESC&start=0&tag=385).

**Comment by MrWilsonWilson** [comment]
you can modify in the cpp to be blueprint read write, ive done it this way and gotten a photo mode to work with it

**Comment by xkamronx** [comment]
I'm also trying to figure out how to modify the FOV at runtime for player options, as basically every player asks about this feature.
